test/aave-v2/TestGovernance.t.sol

```diff
@@ -440,4 +442,18 @@ contract TestGovernance is TestSetup {
     function testFailCallIncreaseP2PDeltasFromImplementation() public {
         exitPositionsManager.increaseP2PDeltasLogic(aDai, 0);
     }
+
+    function testDeprecateCycle() public {
+        hevm.expectRevert(abi.encodeWithSignature("BorrowNotPaused()"));
+        morpho.setIsDeprecated(aDai, true);
+
+        morpho.setIsBorrowPaused(aDai, true);
+        morpho.setIsDeprecated(aDai, true);
+
+        hevm.expectRevert(abi.encodeWithSignature("MarketIsDeprecated()"));
+        morpho.setIsBorrowPaused(aDai, false);
+
+        morpho.setIsDeprecated(aDai, false);
```

Comment:
Should this pass in `true`? Seems like we want to test unpausing a deprecated market.

Reply:
Hmm no because you can't unpause a deprecated market (test failure just above), so we now try to undeprecate it and unpause

Reply:
Ahhh, I see. I wasn't familiar with how `hevm.expectRevert` works, and it seems it checks that the immediate invocation following it will revert. I was thinking it was either, the invocation before the next `hevm.expectRevert` call, or, the last call in the test.

Test looks great :)